src/reference/config/catalogs/catalog-karapace.md

```diff
@@ -1,21 +1,21 @@
 ---
-shortTitle: schema-registry
-description: Zilla runtime schema-registry catalog
```

Comment:
Note: `schema-registry` catalog remains, just in `zilla-plus` now, so `karapace` catalog is net new.